ludwig/utils/data_utils.py

```diff
@@ -159,6 +159,14 @@ def wrapped_fn(*args, **kwargs):
     return wrapped_fn
 
 
+def inner_read(fp, read_fn, **kwargs):
+    if "get_preview" in kwargs:
+        chunksize = kwargs["get_preview"]
+        del kwargs["get_preview"]
+        return next(read_fn(fp, chunksize=chunksize, **kwargs))
```

Comment:
This will only ever return the first chunk. Just double-checking, is that the intended behavior?